CONTRIBUTING.md

```diff
@@ -15,8 +15,8 @@ DATABASE_URL= # Your MongoDB database URL (you can use a local database or a cre
 NEXTAUTH_JWT_SECRET= # random string
 NEXTAUTH_SECRET= # random string
 ```
-
-4. Run `npm run dev` to start the development server.
+4. Run `npx prisma db push` to migrate the database. (Required only for the first time)
```

Comment:
I am not sure if we need the step for init. I have just run for test fresh database and it works fine without that commend